pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.airlift.version>0.148</dep.airlift.version>
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.29</dep.slice.version>
-        <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
+        <dep.aws-sdk.version>1.11.165</dep.aws-sdk.version>
```

Comment:
I was a little bit afraid of taking the latest release, so I took a month old one. Thinking it will be more stable one. Also, from 165 to the latest release, there are no S3 specific improvements.
https://github.com/aws/aws-sdk-java/blob/master/CHANGELOG.md